data/telegraf_plugins.yml

```diff
@@ -2548,6 +2572,15 @@ output:
     introduced: 0.1.9
     tags: [linux, macos, windows, data-stores]
 
+  - name: Parquet
+    id: parquet
+    description: |
+      This plugin writes metrics to parquet files. By default, the parquet
+      output groups metrics by metric name and write those metrics all to the
+      same file. If a metric schema does not match then metrics are dropped.
```

Comment:
```suggestion
      This plugin writes metrics to parquet files. By default, the parquet
      output groups metrics by metric name, and then writes the grouped metrics to a single file. If a metric schema doesn't match, the metrics are dropped.
```

Reply:
Not sure if the meaning is preserved. The plugin creates one file _per group_ and then writes all metrics belonging to this group to the respective file...

Reply:
> Not sure if the meaning is preserved. The plugin creates one file _per group_ and then writes all metrics belonging to this group to the respective file...

Got it - updated.